samples/csharp/end-to-end-apps/Recommendation-MovieRecommender/MovieRecommender_Model/Program.cs

```diff
@@ -14,9 +14,16 @@ namespace MovieRecommenderModel
 
     class Program
     {
-        private static string TrainingDataLocation = @".\Data\ratings_train.csv";
-        private static string TestDataLocation = @".\Data\ratings_test.csv";
-        private static string ModelPath = @"..\..\..\Model\model.zip";
+        private static string BaseModelRelativePath = @"..\..\..\Model";
```

Comment:
@CESARDELATORRE while defining path, the symbols "../" or "..\" represent same meaning ...Am I right?

Reply:
Should be '/' instead '\' so it also works in MacOS/Linux. 
'\' might have issues in Linux/Mac.